source/_components/switch.fritzdect.markdown

```diff
@@ -0,0 +1,43 @@
+---
+layout: page
+title: "AVM FRITZ!DECT Switch"
+description: "Instructions how to integrate your AVM FRITZ!DECT switches into Home Assistant."
+date: 2017-01-24 21:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: avm.png
+ha_category: Switch
+ha_iot_class: "Local Polling"
+ha_release: 0.36.1
```

Comment:
Updated the file (due to a comment in the non-doc PR) and also set the version to 0.37.